iap_mail_domain_blacklist/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows adding domains to the mail domain blacklist to prevent Odoo from assuming that users with the same domain belong to the same team.
```

Comment:
```suggestion
This module allows adding domains to the mail domain blacklist to prevent Odoo from assuming that users with the same domain belong to the same organization.

Due to the nature of this module, in a multi-tenant setting, the configuration in one database affects all databases in the Odoo instance.
```